datasource/src/test/scala/quasar/physical/s3/S3DatasourceSpec.scala

```diff
@@ -151,6 +151,14 @@ class S3DatasourceSpec extends DatasourceSpec[IO, Stream[IO, ?], ResourcePathTyp
         "[[1, 2], [3, 4]]\n".getBytes(Charset.forName("UTF-8")))
     }
 
+//    "read CSV" >>* {
```

Comment:
Leftover comment?

Reply:
Oops! I totally forgot about this case, I need to put a csv file to this bucket and actually check output content.